<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(107, 167, 154);</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>521</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="MailLayout">
     <item>
      <widget class="QListWidget" name="Talks">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 251, 247);</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="ChatBox">
       <item>
        <widget class="QPlainTextEdit" name="writted">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 251, 247);</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="msg">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 251, 247);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="RoomUsers">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>2000</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(246, 251, 255);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="send">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>522</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuChannels">
    <property name="title">
     <string>Channels</string>
    </property>
    <addaction name="actionJoin"/>
   </widget>
   <widget class="QMenu" name="menuUsers">
    <property name="title">
     <string>Users</string>
    </property>
    <addaction name="actionStart_PM"/>
   </widget>
   <widget class="QMenu" name="menuServer">
    <property name="title">
     <string>Server</string>
    </property>
    <addaction name="actionJoinServer"/>
    <addaction name="actionDisconnect"/>
    <addaction name="actionRegister"/>
   </widget>
   <addaction name="menuChannels"/>
   <addaction name="menuUsers"/>
   <addaction name="menuServer"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionJoinServer"/>
   <addaction name="actionStart_PM"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionJoin">
   <property name="icon">
    <iconset theme="icons">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Join</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Join to channel&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
  <action name="actionStart_PM">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/main/icons/key.png</normaloff>:/main/icons/key.png</iconset>
   </property>
   <property name="text">
    <string>Start PM</string>
   </property>
   <property name="toolTip">
    <string>Start private talking</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionJoinServer">
   <property name="icon">
    <iconset theme="icons" resource="icons.qrc">
     <normaloff>:/main/icons/icon_serv.png</normaloff>:/main/icons/icon_serv.png</iconset>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
   <property name="toolTip">
    <string>Connect to server</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconnect From server</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionRegister">
   <property name="text">
    <string>Register</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
